content/courses/onchain-development/anchor-cpi.md

```diff
@@ -354,29 +356,18 @@ on the `init-if-needed` feature take a look at the
 
 ```rust
 [dependencies]
-anchor-lang = { version = "0.25.0", features = ["init-if-needed"] }
-anchor-spl = "0.25.0"
+anchor-lang = { version = "0.30.1", features = ["init-if-needed"] }
```

Comment:
does init if needed work by default now?

Reply:
Just double checked now with anchor 0.30.1, and it throws the following: "init_if_needed requires that anchor-lang be imported with the init-if-needed cargo feature enabled (...)"